integration-libs/opf/checkout/components/opf-checkout-payment-and-review/opf-checkout-payment-and-review.component.ts

```diff
@@ -66,11 +73,14 @@ export class OpfCheckoutPaymentAndReviewComponent
     return Boolean(this.checkoutSubmitForm.get('termsAndConditions')?.value);
   }
 
-  get paymentType$(): Observable<PaymentType | undefined> {
```

Comment:
Is this removal a breaking change?

Reply:
As far as I know, we don't need to worry about breaking changes, since we don't follow major/minor release versions. However, introducing B2B OPF Checkout also impacts B2C components of the OPF, so customers need to be aware that they'll have to adjust their code to use the new OPF functionalities.